src/gui/signal_view_settings.cpp

```diff
@@ -15,7 +15,8 @@ SignalViewSettings::SignalViewSettings (ChannelManager const& channel_manager)
       channel_heigth_in_pixels_ (20),
       channel_manager_ (channel_manager),
       enable_x_grid_ (true),
-      enable_y_grid_ (true)
+      enable_y_grid_ (true),
+      enable_boarder_lines (true)
```

Comment:
LOL... Can't believe this :) 

I am sorry I need to go to bed early tonight... Meeting Paul tomorrow morning. Will keeping working on it tomorrow :)